scripts/in_container/run_install_and_test_provider_packages.sh

```diff
@@ -53,11 +53,20 @@ elif [[ ${INSTALL_AIRFLOW_VERSION} == "wheel"  ]]; then
     echo
     uninstall_airflow_and_providers
     install_airflow_from_wheel "[all]"
+    uninstall_providers
```

Comment:
I also noticed that I did not uninstall providers here. It had no final effect before release (becuase it instlled rc1s first and then overwrote the installations with 1.0.0 versions built locally, but it is cleaner to remove the providers here